include/os/freebsd/spl/sys/simd_x86.h

```diff
@@ -45,6 +45,10 @@
 		fpu_kern_enter(curthread, NULL, FPU_KERN_NOCTX);\
 }
 
+#ifndef PCB_FPUNOSAVE
+#define PCB_FPUNOSAVE PCB_NPXNOSAVE
```

Comment:
Checkstyle says tab, so we should use it.

Reply:
Fixing that